Description: fix build-failure on most Debian architectures:
 SSLeay.xs: In function 'XS_Net__SSLeay_OCSP_response_results':
 SSLeay.xs:5602:3: error: format not a string literal and no format arguments
 [-Werror=format-security]
    croak(error);
    ^
 cc1: some warnings being treated as errors
 .
 Cf. https://buildd.debian.org/status/logs.php?pkg=libnet-ssleay-perl&ver=1.59-1
Origin: vendor
Bug: https://rt.cpan.org/Ticket/Display.html?id=95527
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=95527
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2014-05-10

--- a/SSLeay.xs
+++ b/SSLeay.xs
@@ -5599,7 +5599,7 @@
 	    if (certid) OCSP_CERTID_free(certid);
 	    if (error && !want_array) {
 		OCSP_BASICRESP_free(bsr);
-		croak(error);
+		croak("%s", error);
 	    }
 	}
 	if (!want_array)
